site/plugins.md

```diff
@@ -13,17 +14,21 @@ With a Scope probe plugin, you can insert custom metrics into Scope and have the
 
 ![Custom Metrics With Plugins](images/plugin-features.png)
 
-You can find some examples in [the example plugins](https://github.com/weaveworks/scope/tree/master/examples/plugins) directory.
+## <a id="official-plugins"></a>Official Plugins
 
-There are currently two different examples:
+You can find all the official plugins at [Weaveworks Plugins](https://github.com/weaveworks-plugins).
 
```

Comment:
Official plugins can be found at [Weaveworks Plugins](https://github.com/weaveworks-plugins)